### ISSUE TITLE
/eth/v1/validator/register_validator not implemented

### ISSUE THREAD

**Issue by mcdee** [status-im/nim-beacon-chain#4051]
**Describe the bug**

The `/eth/v1/validator/register_validator` endpoint is not implemented in nimbus.  This is a required API endpoint as per the beacon API spec at https://ethereum.github.io/beacon-APIs/#/Validator/registerValidator

**To Reproduce**

```
curl -w "%{http_code}" -H 'content-type: application/json' --data '[{"message":{"fee_recipient":"0xb943c2c22b1b186a34f47c4dbe2fe367de9ec180","gas_limit":"40000000","timestamp":"1661879190","pubkey":"0xa37b7bb9c412b8cc318fabf7b1fec33eb9634680687f07b977393180ce99889dbcfda81900f3afb9f2281930cf49f5d8"},"signature":"0xa493085fab365d13bea2376434abc3dbfba00a576276c853acabd7b9cb2f2b4b0a90738dd9baeaef75d0f42fa94119a70a09b0ed38fbebb6dde92c9ca062447018821f36c19d6fe34eb8c357d62e5d33e5c1d35035472ef7dd22a7425cdba0c5"}]' http://localhost:5052/eth/v1/validator/register_validator
404
```

This is with nimbus v22.8.1-eaa01d-stateofus

**Comment by jclapis** [comment]
I'll leave my support for this issue here too, as split BN / VC mode won't be able to use MEV-boost without it.

**Comment by zah** [comment]
This has been implemented in the merged PR above and will be shipped in Nimbus 22.9.1

**Comment by mcdee** [comment]
> This has been implemented in the merged PR above and shipped in Nimbus 22.9.0

22.9.0 was released 11 days ago, that PR was from 7 days ago.  Is there a 22.9.1 expected?

**Comment by zah** [comment]
Apologies. Indeed, this will be shipped in 2.9.1 which is expected soon (I've edited my message above).
